python/tvm/relay/frontend/tensorflow.py

```diff
@@ -1931,30 +1931,37 @@ def _impl(inputs, attr, params, mod):
     return _impl
 
 def _partitioned_call():
-    def _impl(inputs, attr, params, mod, graph):
+    from tensorflow.python.framework import function_def_to_graph
+    from tensorflow.python.framework import ops
```

Comment:
Please use try, except blocks here to import these modules..refer other such examples in the same file.